src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler.cs

```diff
@@ -619,6 +619,7 @@ private static int GetBraceInsertionPosition(SyntaxNode node)
                 TryStatementSyntax tryStatementNode => tryStatementNode.TryKeyword.Span.End,
                 CatchClauseSyntax catchClauseNode => catchClauseNode.Block.SpanStart,
                 FinallyClauseSyntax finallyClauseNode => finallyClauseNode.Block.SpanStart,
+                CheckedStatementSyntax checkedStatementNode => checkedStatementNode.Keyword.Span.End,
```

Comment:
does this also include unchecked?

Reply:
Yes, I think checked statement includes both, with the different keyword token

Reply:
Tests added